buildscript.txt

```diff
@@ -1,9 +1,8 @@
 chat
---node-version=6.14.1
---script-version=1.1.10
---build-target=native
---kube=false
---dependencies=redis,mongo
 --language=es
---docker-repos=quay.io/sharelatex
+--node-version=6.14.1
 --acceptance-creds=None
+--dependencies=mongo,redis
```

Comment:
does the buildscript script not keep the dependencies in the same order? would be nice if not so that there aren't extra lines changed when they don't need to be

Reply:
it will keep them in order going forward I think.